source/common/grpc/json_transcoder_filter.cc

```diff
@@ -70,6 +72,14 @@ class TranscoderImpl : public Transcoder {
   std::unique_ptr<TranscoderInputStream> response_stream_;
 };
 
+bool isGrpcResponse(const Http::HeaderMap& headers) {
+  if (Http::Utility::getResponseStatus(headers) != 200) {
+    return false;
+  }
+  return headers.ContentType() &&
```

Comment:
Should this use https://github.com/envoyproxy/envoy/blob/master/source/common/grpc/common.cc#L25? Maybe this util belongs in `Grpc::Common` as well.